frontend_react/src/App.js

```diff
@@ -1,20 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { About, Footer, Header, Skills, Testimonials, Work } from './container';
 import { Navbar } from './components';
 
 function App() {
+  const [theme, setTheme] = useState('light');
+
+  const toggleTheme = () => {
+    if (theme === 'light') {
+      setTheme('dark');
+      document.body.classList.remove('bg-white', 'text-gray-900');
+      document.body.classList.add('bg-gray-900', 'text-white');
+    } else {
+      setTheme('light');
+      document.body.classList.remove('bg-gray-900', 'text-white');
+      document.body.classList.add('bg-white', 'text-gray-900');
+    }
+  };
```

Comment:
Instead of managing the state locally and changing the content directly with DOM APIs or using the manual check, you can keep the state changes in localstorage and make changes with dark: of tailwindcss.

Refer this [guide](https://tailwindcss.com/docs/dark-mode) for the dark mode implementation